examples/basic/README.md

```diff
@@ -16,7 +16,7 @@ uhlive
 Try the example with:
 
 ```text
-npm ci
+npm install
```

Comment:
It is recommanded that `npm ci` should be used when a projet has a `package-lock.json` file, because it will install the exact versions listed in this file. `npm install` will install the last version matching the semver version available in the `package.json` file.

Reply:
In this example folder there is no `package-lock.json`, therefore we should either provide one or update the README.
Could you please tell me which option would be the best in this case?

Reply:
Ok, let's use `npm install`.
We have a special case here: we can't use the `package-lock.json` file because the package version isn't published yet to the npm registry...